Xamarin.Forms.Core/Brush.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Xamarin.Forms
+{
+	public abstract class Brush : BindableObject
+	{
+		public static Brush Default
+		{
+			get { return new SolidColorBrush(Color.Default); }
+		}
+
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public bool IsEmpty
+		{
+			get
+			{
+				Type type = GetType();
+
+				if (type == typeof(SolidColorBrush))
+				{
+					var solidColorBrush = (SolidColorBrush)this;
+					return solidColorBrush == null || solidColorBrush.Color.IsDefault;
+				}
+
+				if (type == typeof(LinearGradientBrush))
+				{
+					var linearGradientBrush = (LinearGradientBrush)this;
+					return linearGradientBrush == null || linearGradientBrush.GradientStops.Count == 0;
+				}
+
+				if (type == typeof(RadialGradientBrush))
+				{
+					var radialGradientBrush = (RadialGradientBrush)this;
+					return radialGradientBrush == null || radialGradientBrush.GradientStops.Count == 0;
+				}
+
+				return true;
+			}
+		}
+
+		public static readonly SolidColorBrush AliceBlue = new SolidColorBrush(Color.AliceBlue);
```

Comment:
Only if we want to allow creating a SolidColorBrush like this:
`SolidColorBrush brush = Brush.Red;`

Instead of:
`SolidColorBrush brush = new SolidColorBrush(Color.Red);`

Reply:
👍